### DISCUSSION TITLE
Pot | 百度 OCR

### DISCUSSION THREAD

**Discussion by giscus** [pot-app/pot-desktop#906] [General]
# Pot | 百度 OCR

跨平台划词翻译和OCR软件

https://pot-app.com/docs/api/recognize/baidu.html

**Comment by jyz2012** [comment]
ee

**Comment by Wesser-i** [comment]
照着上述方法试完之后显示测试失败啊，为啥，，

**Reply by Wesser-i** [reply]
可以了，接口申请下来有10分钟等待时间，过了10分钟后API接口就可用了

**Comment by Luen358** [comment]
一个月2000条，第二个月就要收费了。哎。。。
